src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -222,7 +247,8 @@ protected function getDefinedElements()
      */
     private function getCountryNameOrDefault($code)
     {
-        $countryName = null === $code ? 'Select' : Intl::getRegionBundle()->getCountryNames('en')[$code];
+        $displayLocale = $this->localeContext->getLocaleCode();
```

Comment:
We should not depend on context here, the locale should be passed as an argument.